bridge_adaptivity/docker-compose-stage.yml

```diff
@@ -1,62 +1,8 @@
-version: '2'
+version: '3.4'
 services:
-  postgres:
-    container_name: postgresql_BFA
-    image: postgres
-    env_file: ./envs/pg.env
-    volumes:
-      - pgs:/var/lib/postgresql/data/
-    ports:
-      - "5432:5432"
 
   bridge:
-    container_name: BFA
-    build:
-      context: .
-      dockerfile: Dockerfile_stage
-    image: bridge_adaptivity
-    command: bash -c "./prod_run.sh"
-    volumes:
-      - .:/bridge_adaptivity
-      - static:/www/static
-    ports:
-      - "8000:8000"
-    links:
-      - postgres
-  # Celery worker
-  worker:
```

Comment:
Clarify please why worker and rabbit are removed from the stage docker-compose file?

Reply:
This version of configuring and running Bridge has the make commands for run prod (`make run`), stage (`make run-stage`), and local (`make run-local`) version.  For stage the make command looks like: 
`docker-compose -f docker-compose.yml -f docker-compose-stage.yml up -d`

The prod docker-compose has the main configuration. The stage docker-compose has a specific configuration for the stage. If you run this command: 
`docker-compose -f docker-compose.yml -f docker-compose-stage.yml config`
You can see the docker-compose configuration for the stage. And this configuration has the rabbit and the worker configuration.